docs/root/start/install.rst

```diff
@@ -13,48 +13,47 @@ getting your Envoy proxy up and running.
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can `install Envoy on Debian <https://www.getenvoy.io/install/envoy/debian/>`_
-using `Get Envoy <https://www.getenvoy.io/>`__ until `official packages exist <https://github.com/envoyproxy/envoy/issues/16867>`_.
+You can `install Envoy on Debian using Get Envoy <https://cloudsmith.io/~tetrate/repos/getenvoy-deb-stable/setup/#formats-deb>`_
+until `official packages exist <https://github.com/envoyproxy/envoy/issues/16867>`_.
 
 .. code-block:: console
 
-   $ sudo apt update
-   $ sudo apt install apt-transport-https ca-certificates curl gnupg2 software-properties-common
-   $ curl -sL 'https://getenvoy.io/gpg' | sudo gpg --dearmor -o /usr/share/keyrings/getenvoy-keyring.gpg
-   # Verify the keyring - this should yield "OK"
-   $ echo 1a2f6152efc6cc39e384fb869cdf3cc3e4e1ac68f4ad8f8f114a7c58bb0bea01 /usr/share/keyrings/getenvoy-keyring.gpg | sha256sum --check
-   $ echo "deb [arch=amd64 signed-by=/usr/share/keyrings/getenvoy-keyring.gpg] https://dl.bintray.com/tetrate/getenvoy-deb $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/getenvoy.list
-   $ sudo apt update
-   $ sudo apt install getenvoy-envoy
+   $ apt-get update
+   $ apt-get install -y debian-keyring debian-archive-keyring apt-transport-https curl
+   $ curl -1sLf 'https://deb.dl.getenvoy.io/public/gpg.8115BA8E629CC074.key' | apt-key add -
+   $ curl -1sLf 'https://deb.dl.getenvoy.io/public/config.deb.txt?distro=ubuntu&codename=xenial' > /etc/apt/sources.list.d/tetrate-getenvoy-deb-stable.list
+   $ apt-get update
+   $ apt install -y getenvoy-envoy
 
 Install Envoy on Ubuntu Linux
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can `install Envoy on Ubuntu <https://www.getenvoy.io/install/envoy/ubuntu/>`_
-using `Get Envoy <https://www.getenvoy.io/>`__ until `official packages exist <https://github.com/envoyproxy/envoy/issues/16867>`_.
+You can `install Envoy on Ubuntu using Get Envoy <https://cloudsmith.io/~tetrate/repos/getenvoy-deb-stable/setup/#formats-deb>`_
+until `official packages exist <https://github.com/envoyproxy/envoy/issues/16867>`_.
 
 .. code-block:: console
 
-   $ sudo apt update
-   $ sudo apt install apt-transport-https ca-certificates curl gnupg-agent software-properties-common
-   $ curl -sL 'https://getenvoy.io/gpg' | sudo gpg --dearmor -o /usr/share/keyrings/getenvoy-keyring.gpg
-   # Verify the keyring - this should yield "OK"
-   $ echo 1a2f6152efc6cc39e384fb869cdf3cc3e4e1ac68f4ad8f8f114a7c58bb0bea01 /usr/share/keyrings/getenvoy-keyring.gpg | sha256sum --check
-   $ echo "deb [arch=amd64 signed-by=/usr/share/keyrings/getenvoy-keyring.gpg] https://dl.bintray.com/tetrate/getenvoy-deb $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/getenvoy.list
-   $ sudo apt update
-   $ sudo apt install -y getenvoy-envoy
+   $ apt-get update
+   $ apt-get install -y apt-transport-https gnupg2 curl
+   $ curl -1sLf 'https://deb.dl.getenvoy.io/public/gpg.8115BA8E629CC074.key' | apt-key add -
```

Comment:
the current way of doing this with `deb [signed-by=...]` restricts this key to signing this one archive

Reply:
feel free to click the suggestion button and apply it or otherwise edit the branch. I'll be AFK for a week and won't have time to look up the implications of what you are saying figure out the translation into a diff then apply it.

Reply:
@arkodg I just added you as a collab on my fork in case ^^ doesn't happen. Feel free to push anything.

Reply:
guess this didn't happen which is a shame as I heard people reported known problems. I will follow-up on this personally today